src/toHex.ts

```diff
@@ -6,7 +6,18 @@ import guard from './guard';
  */
 function toHex(color: string): string {
   const [r, g, b, a] = parseToRgba(color);
-  const hex = (x: number) => guard(0, 255, x).toString(16).padStart(2, '0');
+
+  let hex = (x: number) => {
+    const h = guard(0, 255, x).toString(16).padStart(2, '0');
+    // NOTE: padStart could be used here but it breaks Node 6 compat
+    // https://github.com/ricokahler/color2k/issues/351
+    let prefix = '';
+    for (let i = 0; i < 2 - h.length; i++) {
+      prefix += '0';
+    }
```

Comment:
i know this was the solution i opted for in the previous PR however since this is only padding it by one character i think this would be a simpler solution:

```ts
const hex = (x: number) => {
  const h = guard(0, 255, x).toString(16);
  return h.length === 1 ? `0${h}` : h;
};
```